cfme/containers/provider/__init__.py

```diff
@@ -564,6 +572,14 @@ class ContainerObjectAllBaseView(ProvidersView):
     policy = Dropdown('Policy')
     download = Dropdown('Download')
     toolbar = View.nested(ProviderToolBar)
+    SUMMARY_TEXT = None
+
+    @property
+    def summary_text(self):
+        if isinstance(self.SUMMARY_TEXT, string_types):
```

Comment:
Handle when its None cleanly (unlikely since this is a base view, but possible) by:

`if isinstance(self.SUMMARY_TEXT, (string_types, None))`